samples/helloworld/ext2rootfs/Makefile

```diff
@@ -28,7 +28,9 @@ package.pem:
 
 package: package.pem ext2rootfs
 	echo "Generating a signed package"
-	@myst package-sgx --roothash=roothash appdir package.pem ../config.json
```

Comment:
Not sure why this PR has changes that are already merged into main.

Reply:
I had cherry-picked your commit to include your changes, PR was merged at 10PM last night. Will rebase